cypress/support/step-definitions/common-steps.js

```diff
@@ -195,6 +195,10 @@ When('I open component preview', () => {
   commonButtonPreview().click();
 });
 
+When('I open component preview in Docs', () => {
+  commonButtonPreviewDocs().click();
```

Comment:
remove `commonButtonPreviewDocs` import and locator from `index.js` file - you needed it only here

Reply:
👍